frontend/apps/reader/modules/readerbookmark.lua

```diff
@@ -586,4 +586,8 @@ function ReaderBookmark:getLatestBookmark()
     return latest_bookmark
 end
 
+function ReaderBookmark:hasBookmarks()
+    return #self.bookmarks > 0
```

Comment:
I just wrote no further comments, but of course just after writing it occurs to me… could it ever be undefined?

Reply:
Shouldn't have happened with the orders things happen. But might as well check for that, so done :)